File: BackEnd/QuezalliApi/QuezalliApi/Models/Pedido.cs
using System;
using System.Collections.Generic;

namespace QuezalliApi.Models
{
    public partial class Pedido
    {
        public int IdPedidos { get; set; }
        public int IdCliente { get; set; }
        public DateTime FechaPedido { get; set; }
        public DateTime? FechaEntrega { get; set; }
        public int Codigo { get; set; }
        public int Estatus { get; set; }
    }
}

File: FrontEnd/ProyectoQuetzalli/src/app/Interfaces/pedido.ts
export interface Pedido {
    "idPedidos": number,
    "idCliente": number,
    "fechaPedido": string,
    "fechaEntrega": string,
    "total": number,
    "estatus": number
}

export interface Pedido2 {
    "idPedidos": number,
    "idCliente": number,
    "fechaPedido": string,
    "fechaEntrega": string,
    "codigo": number,
    "estatus": number,
    "colonia": string,
    "calle": string,
    "nE": string,
    "nI": string,
    "CP": number
    
}